content/applications/sales/crm/performance/expected_revenue_report.rst

```diff
@@ -0,0 +1,200 @@
+   include leads expected to close within a specific time frame.
+ - :ref:`Exclude unassigned leads <expected_revenue_report/unassigned-leads>`: excludes leads
+   without an assigned salesperson.
+ - :ref:`Specific sales teams <expected_revenue_report/sales-team>`: limits results to only include
+   leads assigned to one or more sales teams. This filter is optional and should not be included if
+   the report is intended for the entire company.
+
+.. _expected_revenue_report/closing-date:
+
+Add filter for expected closing date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the :guilabel:`Add Custom Filter` pop-up window, click into the first field of the new rule.
+Type `Expected Closing` into the :guilabel:`Search...` bar, or scroll to select it from the list.
+Click in the second field and select :guilabel:`is set`. This limits the results to only include
+leads where an estimated closing date is listed.
+
+Next, click the :guilabel:`➕ (plus)` icon to the right of the rule to duplicate it.
+
+.. tip::
+   Using the :guilabel:`➕ (plus)` icon makes it easy to add multiple rules based on the same
+   filter.
+
+In the second field of the new rule, select :guilabel:`is between` from the drop-down menu. This
+creates a set time frame during which the expected closing date must occur for leads to be included
+in the results.
+
+Click in the date fields, and use the calendar drop-down to add a start and end date. This is
+usually the beginning and ending of the current month, or fiscal quarter.
+
+.. _expected_revenue_report/unassigned-leads:
+
+Exclude unassigned leads
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+After filtering for the closing date, add a :guilabel:`New Rule`. Then, click into the new rule's
+first field, and type `Salesperson` in the :guilabel:`Search...` bar, or scroll through the list to
+select it. Click in the rule's second field and select :guilabel:`is set` from the drop-down menu.
+This excludes any results without an assigned salesperson.
+
+.. _expected_revenue_report/sales-team:
+
+Add a filter for sales teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+   This filter is optional. To view results for the entire company, do **not** add this filter, and
+   continue to :ref:`View results <expected_revenue_report/view-results>`.
+
+To limit the results of the report to one or more sales teams, click :guilabel:`New Rule`. Next,
+click the first field for the new rule, and type `Sales Team` in the :guilabel:`Search...` bar, or
+scroll to search through the list to locate it.
+
+In the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
+operator limits results to the sales teams selected in the next field.
+
+Lastly, in the third field, select the desired sales team from the drop-down menu. Multiple teams
+can be added in this field, where each parameter is treated with an "or" (e.g. "any") operator in
+the search logic.
```

Comment:
suggestion to add more context + highlight the last part as a **tip** so it's super visible

```suggestion
Lastly, click into the third field, and either: make selection(s) from the complete list revealed in
the popover menu, or type the first few characters of the specific sales team's title to quickly
find and select it as a parameter.

.. tip::
   Multiple teams can be added to the `Sales Team` rule, where each parameter is treated with an
   "or" (e.g. "any") operator in the search logic.
```